validation/citrus-validation-json/src/main/java/org/citrusframework/validation/json/JsonElementValidator.java

```diff
@@ -0,0 +1,135 @@
+package org.citrusframework.validation.json;
```

Comment:
please add the missing copyright for 2024. also, all touched classes should be updated to copyright years "up to 2024" (`...-2024`).